get-substrate.sh

```diff
@@ -34,11 +36,121 @@ if [[ "$OSTYPE" == "linux-gnu" ]]; then
 elif [[ "$OSTYPE" == "darwin"* ]]; then
 	echo "Mac OS (Darwin) detected."
 
-	if ! which brew >/dev/null 2>&1; then
+	APP="Xcode Command Line Tools"
+	if xcode-select --install 2>&1 | grep "installed"; then
+		echo -e "Skipping, $APP already installed";
+	else
+		echo -e "Installing $APP ...";
+		xcode-select --install;
+	fi
+
+	APP="RBenv"
+	if ! rbenv 2>&1 | grep "command not found"; then
```

Comment:
If the user has a different locale, this `grep` will not work.
Please use something like: https://stackoverflow.com/questions/592620/how-to-check-if-a-program-exists-from-a-bash-script